bin/bootstrap

```diff
@@ -46,18 +50,33 @@ else
   exit 1
 fi
 
+# Ensure zsh is installed
+echo "Installing zsh..."
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+  # Linux
+  sudo apt-get install zsh
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  # macOS
+  brew install zsh
+fi
+
+# Make zsh the default shell
+echo "Changing the default shell to zsh"
+chsh -s $(which zsh)
+
 stat ~/.ssh/github
 if [[ $? != 0 ]]; then
   echo "Github SSH key not found. Generating..."
-  echo "Enter your the email address you'll use for Github: "
+  echo -en "\n\033[1mEnter your the email address you'll use for Github:\033[0m "
```

Comment:
I figured the bold text would make it easier to see you are supposed to enter something.